altimeter/aws/resource/iam/role.py

```diff
@@ -110,3 +120,29 @@ def get_attached_role_policies(client: BaseClient, role_name: str) -> List[Dict[
         for policy in resp.get("AttachedPolicies", []):
             policies.append(policy)
     return policies
+
+
+def get_embedded_role_policies(client: BaseClient, role_name: str) -> List[Dict[str, Any]]:
+    """Get attached embedded policies"""
+    policies = []
+    paginator = client.get_paginator("list_role_policies")
+    for resp in paginator.paginate(RoleName=role_name):
+        for policy_name in resp.get("PolicyNames", []):
+            policy = get_embedded_role_policy(client, role_name, policy_name)
+            policies.append(policy)
+    return policies
+
+
+def get_embedded_role_policy(
+    client: BaseClient,
+    role_name: str,
+    policy_name: str ) -> Dict[str, str]:
```

Comment:
What's the maximum line width for this project? If it fits, I'd probably put the whole function signature in one line. Otherwise, I'd remove at least the space before `)`.

Reply:
To be honest I don't know, there are a couple of python test files with more than 100 chars but just for initializing variables to long queries,  for the rest it looks like all are under 100, so I will leave as it is, but ofc I will remove the space.